### ISSUE TITLE
Indicate that data is loading

### ISSUE THREAD

**Issue by tobinus** [RadioRevolt/DABelFish#53]
When accessing the web page, there is a gap in time between the page loading and the content being loaded (several seconds on my computer). In this gap, the log appears empty and there appears to be no DAB text currently set.

I think we should tell users that content is being loaded. My suggestion is to have a placeholder («Loading…») either as the shown DABtext or in the log (the dabtext is more obvious).

**Comment by tobinus** [comment]
Good reading about loading indicators: https://www.nngroup.com/articles/progress-indicators/

**Comment by boyeln** [comment]
Yes, this is because the miniature version of the database is loaded in to the browser. We could probably use something like this: https://github.com/pcel/meteor-loading/blob/master/README.md

We might need to tell the router to wait for the subscriptions, not sure.
